src/containers/container.js

```diff
@@ -7,25 +7,52 @@ export default class Container extends React.Component {
   constructor(props) {
     super(props);
     this.handleOnClick = this.handleOnClick.bind(this);
-    this.state = {selectedNoteId: null};
+    this.state = {selectedNoteId: null, notes: [...this.props.data]};
```

Comment:
Using state is property of class instead of in constructor()
Eg: `class A extends Component {
  state = {}
}`